yapapi/log.py

```diff
@@ -214,17 +215,22 @@ def log_event(event: events.Event) -> None:
     if not event_logger.isEnabledFor(loglevel):
         return
 
-    exc_info, _ = event.extract_exc_info()
     descr = event_type_to_string[type(event)]
     msg = "; ".join([descr, *(f"{name} = {value}" for name, value in event.__dict__.items())])
-    event_logger.log(loglevel, msg, exc_info=exc_info)
+
+    if event.exc_info:
+        event_logger.log(loglevel, msg, exc_info=event.exc_info)
```

Comment:
`loglevel = logging.DEBUG` for all events. Maybe we can set a default log level for each event?